src/main/webapp/help-initScript.html

```diff
@@ -31,6 +31,14 @@
         </p>
       </li>
       <li>
+        <b>Windows via SSH</b>
+        <p>
+          You can use this <a target="_blank" href="https://raw.githubusercontent.com/Azure/azure-devops-utils/master/powershell/Jenkins-Windows-Init-Script-SSH.ps1">sample</a>.<br/>
+          Java is <b>required</b>, others are optional.<br/>
+          We recommend to use SSH rather than JNLP. For you need less init codes and get much clearer logs.
+        </p>
+      </li>
+      <li>
         <b>Windows w/JNLP</b>
```

Comment:
Seems Linux with JNLP doesn't really work, please also update the documentation to say it's not supported.